src/generator_servo/generator_servo.f90

```diff
@@ -147,7 +147,11 @@ subroutine update_generator_servo(array1, array2)
       Qshaft = array1(4)
       omegagen = array1(3)
       ! Reference mech. torque
-      mech_Qgref = min(Qgref/generatorvar%eta, generatorvar%max_lss_torque)
+      if(generatorvar%eta==0.0) then
```

Comment:
@pilotmm: Numbers in memory are seldom exact zero. Please change to abs().lt.1.d-3. Please also use the original Fortran notations for logical statements, so use .lt. instead of <